diesel_codegen/src/embed_migrations.rs

```diff
@@ -15,7 +15,7 @@ pub fn derive_embed_migrations(input: syn::MacroInput) -> quote::Tokens {
     }
 
     let options = get_options_from_input(&input.attrs, bug);
-    let migrations_path_opt = options.map(|o| get_option(&o, "migrations_path", bug));
+    let migrations_path_opt = options.as_ref().map(|o| get_option(&o, "migrations_path", bug));
```

Comment:
Nice. I think you can just give it `o` instead of `&o` here (same as other comment).